CHANGELOG.md

```diff
@@ -9,17 +9,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [`x.y.z`] - Unreleased
 
+### Breaking changes:
+- The logic for sending Spatial position updates has been changed, the `PositionUpdateFrequency` and the `PositionDistanceThreshold` have now been replaced in the GDK settings by 4 new variables. To update your project:
+  1. Read the more detailed description of the change, you can find it by searching "Spatial Position Updates are now sent based on a different logic" inside this document.
```

Comment:
These steps to update your project don't really walk you through what to do. I'd say the minimum set of things to tell users is what they need to do to get back to what they had before.
i.e. set the min thresholds (pending any possible name change) to be the values these old settings currently have. Then set the maximum to be something much bigger.